Orchestrator/Prognetics.Saga.Orchestrator/Prognetics.Saga.Orchestrator.csproj

```diff
@@ -28,6 +28,10 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.0" />
```

Comment:
If we want to add logger, please do so as part of separate PR.

Reply:
Its not a specific logger. Its a microsoft abstraction of logging that every modern dotnet logging framework implements. We can create a logging abstraction ourselves but I don't see value in that.